api/challenge-response/schemas/components.yaml

```diff
@@ -10,7 +10,7 @@ components:
           type: string
           format: byte
           description:
-            base64 encoded 32, 48, or 64 bytes random value
+            base64 encoded random value. Must be between 8 and 64 bytes long.
```

Comment:
In README.md you said: which means the specified base64 string must be no
longer than 88 bytes).

Is this not contradicting statement here at line 13

Reply:
> Is this not contradicting statement here at line 13

not really, b64 expands by ~1.37

Reply:
precisely: so here we expect a base64 encoded nonce,  which is already ~1.37 times elongated>

Reply:
The random value must be at most 64 bytes. The encoded form will be longer than the value. But I agree that the wording could be clearer. Will amend.